beginner_source/nlp/advanced_tutorial.py

```diff
@@ -217,7 +217,7 @@ def _forward_alg(self, feats):
     def _get_lstm_features(self, sentence):
         self.hidden = self.init_hidden()
         embeds = self.word_embeds(sentence).view(len(sentence), 1, -1)
-        lstm_out, self.hidden = self.lstm(embeds)
+        lstm_out, self.hidden = self.lstm(embeds, self.hidden)
```

Comment:
Thanks for catching this! I wonder how it worked at all. Documentation says `nn.LSTM` expects `(input, (h_0, c_0))`. I should have raise an error, no?